browser/resources/settings/brave_leo_assistant_page/brave_leo_assistant_browser_proxy.ts

```diff
@@ -9,6 +9,7 @@
   resetLeoData(): void
   getLeoIconVisibility(): Promise<boolean>
   toggleLeoIcon(): void
+  getModels(): any
```

Comment:
this should be typed to `Model[]`. You can define the type in this file and import it in the component TS.

Reply:
done